src/traces/contourcarpet/index.js

```diff
@@ -15,7 +15,7 @@ ContourCarpet.supplyDefaults = require('./defaults');
 ContourCarpet.colorbar = require('../contour/colorbar');
 ContourCarpet.calc = require('./calc');
 ContourCarpet.plot = require('./plot');
-ContourCarpet.style = require('./style');
```

Comment:
https://github.com/plotly/plotly.js/pull/1815/commits/0ff2dc469ece4897eae42eefe88df1aa03601d28
This mock on master:
![screen shot 2017-06-29 at 6 39 10 pm](https://user-images.githubusercontent.com/2678795/27713979-4684b48e-5cfc-11e7-8aba-28a9b4de4e1c.png)
and now:
![screen shot 2017-06-29 at 6 40 20 pm](https://user-images.githubusercontent.com/2678795/27713981-4df5d824-5cfc-11e7-8132-ce61ae3827e5.png)